rpc/ethereum/namespaces/miner/api.go

```diff
@@ -143,10 +143,10 @@ func (api *API) SetEtherbase(etherbase common.Address) bool {
 	// NOTE: If error is encountered on the node, the broadcast will not return an error
 	syncCtx := api.clientCtx.WithBroadcastMode(flags.BroadcastSync)
 	rsp, err := syncCtx.BroadcastTx(txBytes)
-	if err != nil || rsp.Code != 0 {
-		if err == nil {
-			err = errors.New(rsp.RawLog)
-		}
+	if rsp != nil && rsp.Code != 0 {
```

Comment:
&& is correct ?

Reply:
yes, https://github.com/tharsis/ethermint/pull/786/files#r757219885